Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -153,61 +153,64 @@ def bump_formula_pr
 
     requested_spec = :stable
     formula_spec = formula.stable
-    odie "#{formula}: no #{requested_spec} specification found!" unless formula_spec
+    odie "#{formula}: no #{requested_spec} specification found!" if formula_spec.nil?
```

Comment:
Another one I'd suggest just changing to `present?` In general I'd say always use `present?` or `blank?` unless you're wanting to handle `nil` differently to `false`.

Reply:
This is also a former `unless` statement—why use `blank?` if it is not a boolean or `Enumerable`?

Reply:
Yes, sorry.